op-chain-ops/genesis/config.go

```diff
@@ -203,8 +203,10 @@ type DeployConfig struct {
 	// FundDevAccounts configures whether or not to fund the dev accounts. Should only be used
 	// during devnet deployments.
 	FundDevAccounts bool `json:"fundDevAccounts"`
-	// The maximum number of participants in the leader election.
-	LeaderElectionMaxParticipants uint64 `json:"leaderElectionMaxParticipants"`
+	// The maximum number of leaders in the leader election.
+	LeaderElectionNumberOfLeaders uint64 `json:"leaderElectionNumberOfLeaders"`
+	// The number of consecutive slots assigned to each leader
```

Comment:
👍  In theory, I think we hit optimal behavior at a little smaller than `SequencerWindowSize`. Less the configured batcher safety margin. 

However, that will result in long periods in which `RoundRobinLeaderElection`'s `HORIZON` is too small to encompass the span. Which... shouldn't break anything, the timeout will get set as soon as we get within HORIZON of the end of the current span.

Reply:
Should we change the way HORIZON is computed then?